Web-Workshops/Workshop 1 - Introduction to Web/Hands on (Dark-Light-Toggler)/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
@MightyMoin Plzzz add the required comments in all the files for a better understanding of the code.

Reply:
@Cronus1007 I think I have added comments at the typical and required places (mainly in script files), and yes comments are not required in HTML and CSS i guess :)